service/JobRoleService.ts

```diff
@@ -7,4 +7,19 @@ module.exports.getJobRoles = async function() {
     } catch (e) {
         return new Error('Could not get job roles.')
     }
-}
\ No newline at end of file
+}
+
+module.exports.getSpecificationById = async function (id: number) { 
+    try {
+        const URL = process.env.UI_URL || 'http://localhost:8080';
```

Comment:
the base url for axios should already be set up to use the UI_URL, you should be able to remove this line